protos/test_run.proto

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package w3c.wptdashboard;
```

Comment:
Should we bother with the `w3c` prefix? When I first put that in strawman.proto I was under the delusion that we could easily package-path our python and go code accordingly without hassle (which we can't, not without hassle).

Reply:
... if you're wondering, the package path matters (at the very least) because it determines the directory structure where BigQuery schema go (and we'll eventually have scripts that need to ingest them and setup/update schema using the `bq` tool).

Reply:
Removed.